### PR TITLE
omhiredis: rewrite documentation

### PR DESCRIPTION
**Pull request by PascalWithopf**
Module documentation was rewritten in the new style.

### CODE REVIEW

File: source/configuration/modules/omhiredis.rst [hunk 1]
```diff
@@ -1,47 +1,132 @@
...
+
+.. csv-table::
+   :header: "type", "default", "mandatory", "|FmtObsoleteName| directive"
+   :widths: auto
+   :class: parameter-table
+
+   "word", "template", "no", "none"
+
+Mode to run the output action in: "queue" or "publish". If not supplied, the
+original "template" mode is used. Note due to a config parsing bug in 8.13,
```
Comment (deoren) [inline source/configuration/modules/omhiredis.rst]: Suggestion: Use either the `note` or `warning` Sphinx directives here. The former is sufficient, the latter might be more appropriate if 8.13 is still used on current LTS distros.

File: source/configuration/modules/omhiredis.rst [hunk 2]
```diff
@@ -1,47 +1,132 @@
...
+   "binary", "off", "no", "none"
+
+If set to on, use RPUSH instead of LPUSH, if not set or off, use LPUSH.
+
+
+Examples
+========
+
+Example 1
+---------
```
Comment (deoren) [inline source/configuration/modules/omhiredis.rst]: Just below this: 

> Note this mode has problems with strings with spaces in them

Suggestion: Use the `note` or `warning` (if appropriate) Sphinx directives here.

**Comment by PascalWithopf** [comment]
@deoren I have added the note directive at both places.

**Comment by deoren** [comment]
@PascalWithopf Thanks!

**Pull request merged**